src/anshuKJun20/CharacterIndex.java

```diff
@@ -0,0 +1,29 @@
+package anshuKJun20;
+
+public class CharacterIndex {
+	
+ void findIndex(String name) {
+	 for(int index = 0 ;index < name.length();index++) {
+		 int count=0;
+		 char ch = name.charAt(index);
+		 if(name.indexOf(ch)== index) {
+		    for(int i =0;i<name.length();i++) {
+		        if(name.charAt(i)== ch) {
+			      count++;
+			        if(count>1) {
```

Comment:
You have increased the complexity in code, Using one for loop and one if block you can achieve the output. 

Reply:
Done 
![oldassg](https://user-images.githubusercontent.com/66829451/85882963-d1a3c000-b7fd-11ea-8255-5e99379cf188.PNG)